meta/documents/changelog_en.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes for Elastic Export Check24.de
 
+## v1.2.3 (2019-09-20)
+
+### Fixed
+Format **Check24DE Fashion**:
+- If a error occurs while creating or updating a SKU for a variation, that error will now be logged and the variation will be exported anyways.
```

Comment:
Hmm anyways is here I think the wrong word. Select one of these :D https://www.dict.cc/?s=dennoch

Reply:
done